index.html

```diff
@@ -29,16 +32,16 @@ <h2>The Past</h2>
             <h2>Why Did It Have To Be Boxes...</h2>
             
             <div class="boxes">
-                <div class="box">Box 1</div>
-                <div class="box">Box 2</div>
-                <div class="box">Box 3</div>
-                <div class="box">Box 4</div>
-                <div class="box">Box 5</div>
-                <div class="box">Box 6</div>
-                <div class="box">Box 7</div>
-                <div class="box">Box 8</div>
-                <div class="box">Box 9</div>
-                <div class="box">Box 10</div>
+                <div class="box" id="one" >Box 1</div>
```

Comment:
These box divs don't need both the class and id in this context. You could replace `<div class="box" id="one" >Box 1</div>` with `<div class="box one">Box 1</div>`, leaving out the ids altogether. The effect is still the same because of specificity, but the .box class must be higher up in the code than .one, .two, etc. classes.